WordPress/src/androidTest/java/org/wordpress/android/e2e/pages/BlockEditorPage.java

```diff
@@ -75,6 +75,7 @@ public BlockEditorPage addPostSettings(String categoryName, String tagName) {
     }
 
     public void openPostSetting() {
+        waitForElementToBeDisplayed(R.id.toolbar_main);
```

Comment:
Added this wait because, during testing, this step failed once when the test wasn't able to click on the menu item (see: [failing test video](https://console.firebase.google.com/project/api-project-108380595987/testlab/histories/bh.f7bbef01bccfafb8/matrices/9043715604138057465/executions/bs.1f9580d7a97bcbe1/videos)).